dashboard/api.py

```diff
@@ -123,47 +163,57 @@ def get_info_for_course(user, course, user_enrollments, user_certificates):
         run_status = run_statuses[0]
 
     if run_status.status in (CourseRunStatus.NOT_ENROLLED, CourseRunStatus.NOT_PASSED):
-        return format_course_for_dashboard(
-            course.get_next_run(),
-            CourseStatus.OFFERED if course.get_next_run() is not None else CourseStatus.NOT_OFFERED,
-            course
-        )
+        next_run = course.get_next_run()
+        status = CourseStatus.OFFERED if next_run is not None else CourseStatus.NOT_OFFERED
+        course_data['status'] = status
+        if next_run is not None:
+            course_data['runs'].append(format_course_for_dashboard(next_run, status))
     elif run_status.status == CourseRunStatus.GRADE:
-        return format_course_for_dashboard(run_status.course_run, CourseStatus.CURRENT_GRADE, course)
+        course_data['status'] = CourseStatus.CURRENT_GRADE
+        course_data['runs'].append(format_course_for_dashboard(run_status.course_run, CourseStatus.CURRENT_GRADE))
```

Comment:
Can you rename `format_course_for_dashboard` to `format_courserun_for_dashboard`?